package.json

```diff
@@ -20,23 +20,23 @@
   "main": "node/lib/index.js",
   "types": "node/lib/index.d.ts",
   "files": [
-    "npm-scripts.js",
     "node/lib",
+    "npm-scripts.js",
     "worker/Makefile",
-    "worker/meson.build",
-    "worker/meson_options.txt",
-    "worker/src",
+    "worker/deps/libwebrtc",
     "worker/include",
-    "worker/test/src",
-    "worker/test/include",
-    "worker/fuzzer/src",
     "worker/fuzzer/include",
-    "worker/subprojects/*.wrap",
-    "worker/deps/libwebrtc",
+    "worker/fuzzer/src",
+    "worker/meson.build",
+    "worker/meson_options.txt",
+    "worker/scripts/*.js",
+    "worker/scripts/*.json",
     "worker/scripts/*.py",
     "worker/scripts/*.sh",
-    "worker/scripts/*.js",
-    "worker/scripts/*.json"
+    "worker/src",
+    "worker/subprojects/*.wrap",
+    "worker/test/include",
+    "worker/test/src"
```

Comment:
Is it just about sorting items alphabetically? If so, @nazar-pc: is this what you asked for?

Reply:
Yes, only sort them, so it's easier to identify later both additions and deletions.

Reply:
Directories should go first (all of them), then files (starting with those that are capitalized if any). Basically should be the same order you typically see it in the repository or file manager. Just open https://github.com/versatica/mediasoup and see how files are listed.

Reply:
Like this?

```js
    "node/lib",
    "worker/deps/libwebrtc",
    "worker/fuzzer/include",
    "worker/fuzzer/src",
    "worker/include",
    "worker/scripts/*.js",
    "worker/scripts/*.json",
    "worker/scripts/*.py",
    "worker/scripts/*.sh",
    "worker/src",
    "worker/subprojects/*.wrap",
    "worker/test/include",
    "worker/test/src",
    "worker/Makefile",
    "worker/meson.build",
    "worker/meson_options.txt",
    "npm-scripts.js"
```